py/client/pydeephaven/table.py

```diff
@@ -219,6 +220,23 @@ def sort(self, order_by: Union[str, List[str]], order: Union[SortDirection, List
         """
         return super().sort(order_by, order)
 
+    def sort_descending(self, order_by: Union[str, List[str]]) -> Table:
```

Comment:
* Should there be a `sort_ascending`?
* Is this needed if a user can simply do `sort(order_by, DESCENDING)`?
* If this method is here, should it also be on the proxy?

Reply:
- It does seem odd to have it, but I think only someone who remembers its history can explain its existence and the need to keep it for backwards compatibility.
- see above
- not sure if we want to pollute the API when we have doubts about its very existence.

Reply:
I see https://github.com/deephaven/deephaven-core/issues/4083.  The user wants the client API to match the server API, which is reasonable.  Looking at our server API, we only have `sort_descending`.  `sort_ascending` is accomplished via `sort` with default arguments.

Looking some more, the server has this method for:
* PartitionedTableProxy
* RollupNodeOperationsRecorder
* Table
* TreeNodeOperationsRecorder

Looking at the client, we just have Table, so it appears that this is the only needed change.

All that to say that I think the needed work is done.